packages/trpc/server/routers/viewer/dsync/get.handler.ts

```diff
@@ -26,9 +27,23 @@ export const getHandler = async ({ ctx, input }: Options) => {
     });
   }
 
-  const tenant = input.organizationId ? `${tenantPrefix}${input.organizationId}` : (samlTenantID as string);
+  // Right now dsync is only available for organizations
+  if (!input.organizationId) {
+    throw new TRPCError({
+      code: "BAD_REQUEST",
+      message: "Organization ID is required",
+    });
+  }
+
+  const dsyncData = await prisma.dSyncData.findUnique({
+    where: {
+      organizationId: input.organizationId,
+    },
+  });
```

Comment:
Only render the dsync data if it exists in our DB. I was getting hard blocked because it would get the correct dsync data but it wasn't associated with an org.